precompile/wasm/IWasm.sol

```diff
@@ -10,6 +10,54 @@ IWasm constant WASM_CONTRACT = IWasm(
 );
 
 interface IWasm {
+    // Events
+    /**
+     * @dev InstantiateContract defines an event emitted when a wasm contract is successfully
+     * instantiated via instantiateContract or instantiateContract2
+     * @param sender the address of the sender
+     * @param admin the address of the contract admin
+     * @param contractAddress the address of the instantiated contract
+     * @param codeId the id of contract
+     * @param label the label of contract
+     */
+    event InstantiateContract(
+        address indexed sender,
+        address indexed admin,
+        address indexed contractAddress,
+        uint256 codeId,
+        string label
+    );
+
+    /**
+     * @dev ExecuteContract defines an event emitted when a wasm contract is successfully
+     * executed via executeContract
+     * @param sender the address of the sender
+     * @param contractAddress the address of executed contract
+     * @param msg executed message
+     * @param funds the amounts of transferred coin
+     */
+    event ExecuteContract(
+        address indexed sender,
+        address indexed contractAddress,
+        bytes msg,
+        Coin[] funds
+    );
+
+    /**
+     * @dev MigrateContract defines an event emitted when a wasm contract is successfully
+     * migrated via migrateContract
+     * @param sender the address of the sender
+     * @param contractAddress the address of migrated contract
+     * @param codeId changed code id
+     * @param msg the message passed during migration
+     */
+    event MigrateContract(
+        address indexed sender,
+        address indexed contractAddress,
+        uint256 codeId,
+        bytes msg
```

Comment:
It would be nice if response value data was also added.

Reply:
[49cb90b](https://github.com/xpladev/xpla/pull/176/commits/49cb90bb1dbd205e6789a2a59f7024cd06856bd5)